src/main/resources/application.yaml

```diff
@@ -81,6 +81,18 @@ accesstoken:
     - serviceName: finrem
       validity: ${SAS_TOKEN_VALIDITY:300} #In seconds
 
+containers:
+  mappings:
+    - container: sscs
+      jurisdiction: SSCS
+      poBox: 12626
+    - container: probate
+      jurisdiction: PROBATE
+      poBox: 12625
+    - container: divorce
+      jurisdiction: DIVORCE
+      poBox: 12706
```

Comment:
Jira ticket mentions much more data in this field.
Is this all that is sent in the metadata file?

Reply:
@nitinprabhu confirmed that only pobox number is sent in the metadata file

Reply:
👍